screenpipe-app-tauri/components/meeting-history.tsx

```diff
@@ -267,7 +279,9 @@ export default function MeetingHistory() {
         },
         {
           role: "user" as const,
-          content: `${enhancedPrompt}:\n\n${meeting.fullTranscription}`,
+          content: `${enhancedPrompt}:\n\n${meeting.segments
```

Comment:
i think it's important to keep the timestamp (probably in a more human readable format) both for human and machine summary

Reply:
Okay I'll update that

Reply:
<img width="387" alt="image" src="https://github.com/user-attachments/assets/8525edda-d763-498d-b716-478cf7d1c4ee">
eg this but more human readable date and probably should tell the LLM current date and stuff about local / UTC or timezone if useful

Reply:
I'll make sure it is the same format

Reply:
<img width="1312" alt="image" src="https://github.com/user-attachments/assets/8bf5d70e-bf48-4b04-ad5c-743181c17744">